packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -97,6 +98,7 @@ export function executeHandlebarsTemplate(
     },
     cache
   )
+  registerHandlebarHelpers()
```

Comment:
Could we pass `intl` down to the helpers from here so that custom handlebar helpers can localise the values in use?

```ts
export function registerHandlebarHelpers(intl: IntlShape) {
  for (const funcName of Object.keys(handlebarHelpers)) {
    const func = handlebarHelpers[funcName]({ intl })
    if (typeof func === 'function') {
      Handlebars.registerHelper(funcName, func)
    }
  }
}
``` 

So the definition for a function handlebar helper in country config would become

```ts
function myHelper(options: { intl: IntlShape }) {
  return (valueToTransform: string) => valueToTransform
}
```

or just

```ts
function myHelper() {
  return (valueToTransform: string) => valueToTransform
}
```

if intl is not needed

Reply:
I would like to, but going to come back to that in a different PR.  At the moment I am struggling with extra params conflicting with the expected params to Handlebars.HelperDelegate